### DISCUSSION TITLE
[Idea]: Option to have Media Presentation mode as the default view

### DISCUSSION THREAD

**Discussion by JamesSwift** [sircharlo/meeting-media-manager#2415] [Ideas] [answered]
### Is your idea related to a problem?

I'm testing out m3 for use at our meetings, with the goal of trying to simplify things for those assigned to AV. It took me a minute to find the media presentation mode button.

### Describe your idea

Could we have an option to have the media view as the default when the app launches? And possibly alter the buttons/navigation to make it clearer?

### Describe alternatives you've considered

n/a

### Additional context

I'm not sure how the automatic media syncing works, whether it currently requires being on the calendar view for it to complete. But assuming it's happening in the background, I can't see why the calendar would need to be the default view for a single congreation computer. But I'm very new to m3 :-) Thanks for all your work

**Comment by sircharlo** [comment]
A rework if the UI is currently underway. It will greatly simplify the app's interface. By default, the app will open to media presentation view for the current day, if it's a meeting day. Also, it'll be alot better looking 😄 
Unfortunately, our free time is extremely limited these days, but we haven't forgotten about this oft-requested feature. Thanks for your patience!

**Reply by JamesSwift** [reply]
Thank you. I'm happy to be patient.

**Reply by ux-git** [reply]
@sircharlo @mtdvlpr If you need any assistance with the UI design, feel free to ask me. I will be glad to help and collaborate!

**Reply by mtdvlpr** [reply]
@ux-git, thanks for your willing spirit! For now, our goal is to migrate the existing UI and functionality to the new codebase. Once that's complete, we can start improving the UI/UX. Unfortunately, though, both Sircharlo and I scarcely have time to work on this migration currently.

**Reply by ux-git** [reply]
@mtdvlpr sure! I'm in touch in any case 🤝

By the way, I can see that you've been working on the migration for quite some time already. What are your realistic expectations to finish all the refactoring?

**Reply by e-Feitosa** [reply]
The current M3 already works very well, with an update to a new version, I believe it will get even better. We welcome information on the progress of the new version.

**Comment by mtdvlpr** [comment, marked as answer]
Implemented as of `v24.10.6`
